bitnami/influxdb/values.yaml

```
@@ -279,8 +303,56 @@ influxdb:
     ##    cpu: 100m
     ##    memory: 128Mi
     requests: {}
-  ## InfluxDB&trade; pods' liveness probe
-  ## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+  ## @param influxdb.command Override default container command (useful when using custom images)
+  ##
+  command: []
+  ## @param influxdb.args Override default container args (useful when using custom images)
+  ##
+  args: []
+  ## @param influxdb.lifecycleHooks for the InfluxDB&trade; container(s) to automate configuration before or after startup
+  ##
+  lifecycleHooks: {}
+  ## @param influxdb.extraEnvVars Array containing extra env vars to configure InfluxDB&trade;
+  ## For example:
+  ## extraEnvVars:
+  ##  - name: INFLUXDB_DATA_QUERY_LOG_ENABLED
+  ##    value: "true"
+  ##
+  extraEnvVars: []
+  ## @param influxdb.extraEnvVarsCM Name of existing ConfigMap containing extra env vars for InfluxDB&trade; nodes
+  ##
+  extraEnvVarsCM: ""
+  ## @param influxdb.extraEnvVarsSecret Name of existing Secret containing extra env vars for InfluxDB&trade; nodes
+  ##
+  extraEnvVarsSecret: ""
+  ## @param influxdb.extraVolumes Array of extra volumes to be added to the deployment (evaluated as template). Requires setting extraVolumeMounts
+  ##
+  extraVolumes: []
+  ## @param influxdb.extraVolumeMounts Array of extra volume mounts to be added to the container (evaluated as template). Normally used with extraVolumes.
+  ##
+  extraVolumeMounts: []
+  ## @param influxdb.containerPorts [object] Container ports
```

In this way both values are shown in the table
```suggestion
  ## @param influxdb.containerPorts Container ports
```